dev_guide/application_health.adoc

```diff
@@ -44,6 +44,21 @@ Set a readiness check by configuring the
 
 |===
 
+The exact timing of a probe is controlled by two fields,
+both expressed in units of seconds:
+
+|====
+| Field | Meaning
```

Comment:
Minor suggestion:

s/Meaning/Description/

To match similar tables in other topics.